### PR TITLE
android plugin implemented

### PR DESCRIPTION
**Pull request by bijoshtj**

### CODE REVIEW

File: android/FreshchatPlugin.java [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    try {
+      reqJson = new JSONObject(param);
+      Freshchat.getInstance(appContext).identifyUser(reqJson.getString("id"),
+        null);
+    } catch (Exception e){
+      logger.log(TAG + "{exception}", "" + e.getMessage());
+    }
+  }
+
+  public void addMetaData (String param) {
```
Comment (rampr) [inline android/FreshchatPlugin.java:138]: is this for events ?

**Comment by rampr** [comment]
Where is the iOS plugin ?

File: android/FreshchatPlugin.java (hunk 1 above)
Comment (bijoshtj) [reply android/FreshchatPlugin.java:138]: nope. its for adding custom user properties @rampr

**Review by Javasamurai — APPROVED** [review]

**Comment by bijoshtj** [comment]
@Javasamurai  is doing ios plugin. @rampr

**Review by rampr — APPROVED** [review]

**Pull request merged**